packages/ckeditor5-engine/src/index.js

```diff
@@ -16,10 +16,18 @@ export { default as Conversion } from './conversion/conversion';
 
 export { default as HtmlDataProcessor } from './dataprocessor/htmldataprocessor';
 
+export { default as InsertOperation } from './model/operation/insertoperation';
```

Comment:
Why do we need to add these?

Reply:
CF.

Reply:
If we're adding InsertOperation we should also add all other operations here.